src/main/java/org/unbescape/html/HtmlEscapeUtil.java

```diff
@@ -433,9 +437,60 @@ static int translateIllFormedCodepoint(final int codepoint) {
             case 0x9A: return 0x0161;
             case 0x9B: return 0x203A;
             case 0x9C: return 0x0153;
+            case 0x9D: return 0xFFFD;
             case 0x9E: return 0x017E;
             case 0x9F: return 0x0178;
-            default: return codepoint;
+
+            case 0x000B: return 0xFFFD;
```

Comment:
I could move this above so that it fits in with the other two digit replacements if you prefer it to be inline. I separated these replacements out because they are semantically different.